tools/bumper/parser_test.go

```diff
@@ -0,0 +1,185 @@
+				Expect(componentsConfig.Components["multus"].Metadata.Tag).To(Equal("v3.4.1"))
+
+				By("Checking LinuxBridge params are valid")
+				Expect(componentsConfig.Components["linux-bridge"].Url).To(Equal("https://github.com/containernetworking/plugins"))
+				Expect(componentsConfig.Components["linux-bridge"].Commit).To(Equal("ad10b6fa91aacd720f1f9ab94341a97a82a24965"))
+				Expect(componentsConfig.Components["linux-bridge"].Metadata.Tag).To(Equal("v0.8.6"))
+
+				By("Checking BridgeMarker params are valid")
+				Expect(componentsConfig.Components["bridge-marker"].Url).To(Equal("https://github.com/kubevirt/bridge-marker"))
+				Expect(componentsConfig.Components["bridge-marker"].Commit).To(Equal("049eb6a796b742767a0be838f6ddb7f79c97dd60"))
+				Expect(componentsConfig.Components["bridge-marker"].Metadata.Tag).To(Equal("0.3.0"))
+
+				By("Checking Kubemacpool params are valid")
+				Expect(componentsConfig.Components["kubemacpool"].Url).To(Equal("https://github.com/k8snetworkplumbingwg/kubemacpool"))
+				Expect(componentsConfig.Components["kubemacpool"].Commit).To(Equal("7728339b6c8dcfc9d53cc18ce8e06a910094ed47"))
+				Expect(componentsConfig.Components["kubemacpool"].Metadata.Tag).To(Equal("v0.16.0"))
+
+				By("Checking Nmstate params are valid")
+				Expect(componentsConfig.Components["nmstate"].Url).To(Equal("https://github.com/nmstate/kubernetes-nmstate"))
+				Expect(componentsConfig.Components["nmstate"].Commit).To(Equal("6f41747788235c3363a314dfae46a4ed90e1846a"))
+				Expect(componentsConfig.Components["nmstate"].Metadata.Tag).To(Equal("v0.25.0"))
+
+				By("Checking OvsCni params are valid")
+				Expect(componentsConfig.Components["ovs-cni"].Url).To(Equal("https://github.com/kubevirt/ovs-cni"))
+				Expect(componentsConfig.Components["ovs-cni"].Commit).To(Equal("edb4754d08f49be54c399c938895fe82aab6aa5a"))
+				Expect(componentsConfig.Components["ovs-cni"].Metadata.Tag).To(Equal("v0.12.0"))
+
+				By("Checking MacvtapCni params are valid")
+				Expect(componentsConfig.Components["macvtap-cni"].Url).To(Equal("https://github.com/kubevirt/macvtap-cni"))
+				Expect(componentsConfig.Components["macvtap-cni"].Commit).To(Equal("14e5a3c7bf516bd4542366a78b7af02f04230ac5"))
+				Expect(componentsConfig.Components["macvtap-cni"].Metadata.Tag).To(Equal("v0.2.0"))
+			})
+		})
+	})
+
+	Describe("UpdateComponentsYaml function", func() {
+		Context("when yaml file is valid", func() {
+			var componentsConfig *componentsConfig
+			var err error
+
+			BeforeEach(func() {
+				By("Creating a temporary file with empty content")
+				createTempFileWithContent(componentsDummyYaml)
+
+				By("Parsing the yaml")
+				componentsConfig, err = parseComponentsYaml(tmpfile.Name())
+				Expect(err).ToNot(HaveOccurred(), "Should Succeed to parse file")
+			})
+
+			It("should correctly update file", func() {
+				By("Changing some params in the yaml")
+				newComponent := component{Url: "www.google.com", Commit: "12345", Metadata: metadata{Tag: "v1.2.3"}}
```

Comment:
Use fields value that make sense, it's better for testing purposes.

Reply:
ok, I'll also add another test for adding a new component